### DISCUSSION TITLE
how to reactivate rdp after cis hardening

### DISCUSSION THREAD

**Discussion by blackcaty1** [0x6d69636b/windows_hardening#133] [Q&A] [answered]
can't get it to allow an rdp connection again after cis hardening.

Where can I reactivate this in the registry?
Or which item must be removed from the cis list to not block rdp?

Windows: 21h2
list: cis 21h2_machin

**Comment by 0x6d69636b** [comment, marked as answer]
Are you using a standalone computer without a domain? Then it might be the setting _2.2.20 | User Rights Assignment | Deny log on through Remote Desktop Services_, you have to remove the setting _NT AUTHORITY\Local account_

**Reply by blackcaty1** [reply]
yes it is a normal pc , complete clean installation.
I have removed the rule but no connection is possible.
I also removed all the rules that are related to the remote desktop but that did not really help. Now I can activate the switch in the settings for rdp again, but it goes immediately again on off.

**Reply by 0x6d69636b** [reply]
Did you change the setting back to the default value as well? And you should check the setting _18.9.63.3.2.1 | Administrative Templates: Windows Components | Remote Desktop Session Host: Allow users to connect remotely by using Remote Desktop Services_, if this is set to 1 then remote desktop connections are denied.
